examples/Web/Controllers/HomeController.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using Confluent.Kafka;
+
+
+namespace Web.Controllers
+{
+    public class HomeController : Controller
+    {
+        private readonly Random random;
+        private string topic;
+        private readonly KafkaDependentProducer<Null, string> producer;
+
+        public HomeController(KafkaDependentProducer<Null, string> producer, IConfiguration config)
+        {
+            // In a real-world application, you might be using Kafka as a system of record and may wish
+            // to update application state in your request handlers. Or you may wish to write some
+            // analytics data related to your handler logic. In this example, we aren't doing anything
+            // interesting, so we just write frivolous messages to demonstrate how to set things up.

```

Comment:
I had to look up frivolous, so other people will too.

Reply:
hmm. point taken, and agree, but i can't think of anything better and i think people can use a dictionary.